docs/configure-vars-env.md

```diff
@@ -0,0 +1,24 @@
+# Configure my environment
+
+## Copy environment variables
+
+A `.env` file should be created in the repository root, to keep configuration default on the dev setup:
+
+```
+cp example.env .env
+```
+
+Replace `REPO_PATH_SERVER` with the path from above.
```

Comment:
```suggestion
```

Reply:
I changed the instructions in this commit : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/8f0ba7b3cb62654d396885a85090ef1f2d8d3749